src/caduceus/outboundSender.go

```diff
@@ -465,12 +494,12 @@ func (obs *CaduceusOutboundSender) send(secret, acceptType string, msg *wrp.Mess
 	}
 	payloadReader = bytes.NewReader(body)
 
-	req, err := http.NewRequest("POST", obs.id, payloadReader)
+	req, err := http.NewRequest("POST", urls.Value.(string), payloadReader)
```

Comment:
For the first request, we should use the primary url (aka `id`). If that fails then we can leverage the alternative urls.
```suggestion
	req, err := http.NewRequest("POST", obs.id, payloadReader)
```